rnd/autogpt_builder/src/types/api/index.ts

```diff
@@ -0,0 +1,75 @@
+import {XYPosition} from "reactflow";
+import {STATUS_COMPLETED, STATUS_FAILED, STATUS_INCOMPLETE, STATUS_QUEUED, STATUS_RUNNING} from "@/constants/constants";
+
+export type ObjectSchema = {
```

Comment:
@Pwuts this is the kind of structure I would like to have, types/api etc

Reply:
For application types I agree. But imo the API stuff should be a self-contained module, and that includes its types.

Reply:
To get this moving, let's structure it like that for now

so how about keeping the rest of the structure I suggested and pull back the api client and types into a module?

Reply:
Outside of the comments on the API client and types and constants, it looks good to me :)